public/app/features/dashboard-scene/scene/DashboardControls.tsx

```diff
@@ -74,6 +75,20 @@ export class DashboardControls extends SceneObjectBase<DashboardControlsState> {
       refreshPicker: state.refreshPicker ?? new SceneRefreshPicker({}),
       ...state,
     });
+
+    this.addActivationHandler(() => {
+      let refreshPickerUnsub: CancelActivationHandler | undefined;
+
+      if (this.state.hideTimeControls && this.state.refreshPicker.state.refresh) {
+        this.state.refreshPicker.activate();
```

Comment:
this needs to be asigned to `refreshPickerUnsub` i think? :)

Reply:
adjusted and fully working now, sorry was in a bit of a rush. Not sure if this solution is good enough though